tools/nillion/src/serialize.rs

```diff
@@ -37,22 +37,8 @@ pub fn serialize_output(format: &CommandOutputFormat, data: &dyn SerializeAsAny)
 }
 
 pub fn serialize_error(format: &CommandOutputFormat, e: &anyhow::Error) -> String {
-    let main_error = e.to_string();
-    let mut causes: Vec<String> = e.chain().skip(1).map(|cause| cause.to_string()).collect();
-
-    let error = if causes.is_empty() {
-        if let Some((first_part, second_part)) = main_error.split_once(':') {
-            causes.push(second_part.trim().to_string());
-            first_part.trim().to_string()
-        } else {
-            main_error
-        }
-    } else {
-        main_error
-    };
-
-    let cause = (!causes.is_empty()).then_some(causes);
-
-    let error_response = ErrorOutput { error, cause };
+    let error = e.to_string();
+    let causes: Vec<String> = e.chain().skip(1).map(|cause| cause.to_string()).collect();
+    let error_response = ErrorOutput { error, causes };
```

Comment:
I should clarify here: this was trimming the wrong thing, e.g. I had a timestamp which contains `:` and it was splitting it and considering the last part of it a `cause`. I think between trying to guess and not providing any causes I would prefer to not provide any causes and avoid messing up the output.